JSTests/stress/get-by-val-generic-structurestubinfo.js

```diff
@@ -0,0 +1,49 @@
+//@ requireOptions("JSC_getByValICMaxNumberOfIdentifiers=2", "JSC_useConcurrentJIT=0")
```

Comment:
I think you want `--getByValICMaxNumberOfIdentifiers=2`

Reply:
Oops, fixed!